ui/components/app/confirm-page-container/flask/snap-insight.js

```diff
@@ -64,7 +64,9 @@ export const SnapInsight = ({ transaction, chainId, selectedSnap }) => {
                     >
                       {key}
                     </Typography>
-                    <Typography variant={TYPOGRAPHY.H6}>{data[key]}</Typography>
+                    <Typography variant={TYPOGRAPHY.H6}>
+                      {JSON.stringify(data[key])}
```

Comment:
Can we at least make the output a bit more readable?
```suggestion
                      {JSON.stringify(data[key], null, 2)}
```

Reply:
Made it prettier 😄